debsources/app/patches/routes.py

```diff
@@ -37,6 +37,16 @@ def skeleton_variables():
     lambda e: (ErrorHandler()(e, http=404), 404))
 
 
+# Before request
+@bp_patches.before_request
+def before_request():
+    endpoints = ['summary', 'patch_view']
```

Comment:
ping for the point mentioned in irc that happens here as well (so that I don't lose track) :)